src/main/java/com/thevoxelbox/voxelsniper/brush/type/SpiralStaircaseBrush.java

```diff
@@ -12,6 +12,7 @@
 import com.thevoxelbox.voxelsniper.sniper.toolkit.ToolkitProperties;
 import org.bukkit.ChatColor;
 
+import java.util.List;
 import java.util.stream.Stream;
 
 public class SpiralStaircaseBrush extends AbstractBrush {
```

Comment:
How is this brush supposed to work? And why can I only use two types of stair cases?
/b sstair woodstair
![](https://i.imgur.com/AOhQVFk.png)

Reply:
No idea, just ported the original voxel brush 🤷

Reply:
I found this description at some wiki:

> You can use this to create a spiral stair case for you. The arrow will remove any surrounding blocks within the radius you specified and spawn the stair case. You can use the gunpowder tool for this however that tool will not let you go below grass level.

Reply:
Note: Just tested on 1.12 with legacy VS and the outcome is the same. So not something that is newly "broken", dunno if you can call this broken, idk what it's supposed to do, but to my understanding it should create an actual spiral staircase around the targeted block?

Reply:
You must set a voxel height (``/vh [n]``), will add a note, tried with 10, got this build:

![2021-08-12_23 26 55](https://user-images.githubusercontent.com/43724816/129272339-2f9be88e-f486-4515-9d2d-ee988a02469d.png)

Reply:
Good to know, maybe add an arg to pass the width of the staircase? Or is it always one block 👀

Reply:
Probably needs a rewrite to accept all stair types, another pr later tbh, sorry.

Reply:
Nw, bad enough it's hard coded within 30+ occurrences in the SpiralStairCase class